src/openzaak/utils/middleware.py

```diff
@@ -15,3 +33,72 @@ def __call__(self, request):
 
     def log(self, request: HttpRequest):
         logger.debug("Request headers for %s: %r", request.path, request.headers)
+
+
+class JWTAuth(_JWTAuth):
+    def __init__(self, encoded: str = None):
```

Comment:
wouldn't it make more sense to declare `component = None` as a class attribute and not override the `__init__`?

Reply:
fixed